src/slave/slave.cpp

```diff
@@ -7157,11 +7157,6 @@ void Slave::_checkDiskUsage(const Future<double>& usage)
     LOG(ERROR) << "Failed to get disk usage: "
                << (usage.isFailed() ? usage.failure() : "future discarded");
   } else {
-    executorDirectoryMaxAllowedAge = age(usage.get());
```

Comment:
Here `executorDirectoryMaxAllowedAge` is actually a member variable of `Slave` that is used for generating the value of the metric, so removing the assignment will probably break this. (Imho this is also a fault in our C++ style guide, having no visual indication of member variables makes it too easy to make this kind of mistake)